---
title: "Emile Gluck-Thaler ~ Research"
layout: textlay
excerpt: "Emile Gluck-Thaler ~ Research"
sitemap: false
permalink: /research/
---

## Research overview

Genetic variation is the ultimate source of all evolutionary adaptations and innovations.  Traditionally, genetic variation is measured by calculating differences among individuals at single nucleotide sites. Yet it is increasingly apparent that individuals also differ extensively in the content and organization of their genomes. Our understanding of how this "structural" variation impacts evolution is still in its infancy, yet it likely has distinct implications for the emergence and maintenance of adaptive traits. This is because structural mutations often change large amounts of sequence at a time (e.g., insertions and deletions), affect inheritance patterns across alleles and loci (e.g., inversions and rearrangements), and can even jump around within and between genomes (e.g., transposons and other mobile elements). Identifying the mechanisms generating selectable variation in genome structure and their phenotypic consequences is thus crucial for deciphering the genetic bases of adaptation. By investigating differences and similarities in genome structure across populations and species, I aim to answer the following questions:

**1) What mechanisms generate variation in genome content?**

**2) When do demographic and environmental conditions favor the evolution of genome organization?**

**3) How does structural variation constrain or facilitate host-microbe interactions?**

I work primarily with fungi, because not only are their genomes rife with structural variation about which we know very little, but because they are the primary agents of plant disease and plant mutualism. There is therefore an urgent need to understand how plant-fungal interactions evolve so we can develop novel practices for managing fungal threats and opportunities in agro- and forest ecosystems.

## Main research projects

### *Starships* and the mobilization of ecological traits

![]({{ site.url }}{{ site.baseurl }}/images/research/voyager.png){: style="width: 350px; float: right; border: 25px; padding:25px" figcaption}*A 90kb Starship carrying a candidate virulence factor*

While fungal genomes harbour extensive structural variation, its origins have remained obscure. I led a collaborative sequencing initiative to discover a novel group of unusually large mobile elements called *Starships*, [whose activities generate variation in genome content and organization](https://www.biorxiv.org/content/10.1101/2021.12.13.472469v2). *Starships* are endemic in the Pezizomycotina, the largest sub-phylum of fungi, and are found in species with diverse ecologies, from mutualists to lichens to plant and animal pathogens. Many *Starships* carry putatively adaptive host genes as cargo and thus have vast potential to shape ecological outcomes and species interactions. For example, we found that a *Starship* carrying a putative virulence factor in a global fungal plant pathogen is associated with life history trade-offs between pathogenicity and growth. All *Starships* share a single origin, which suggests their activities have been ongoing for hundreds of millions of years. Parallels between *Starships* and bacterial mobile elements further suggest that *Starships* are the first dedicated agents of active gene transfer in eukaryotes, [a mechanism which has long been searched for](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1005156). I am now working to develop software for automating  *Starship* annotation and investigating the genetic mechanisms driving their birth-and-death dynamics within and between species. 

### The architecture of local adaptation in a global plant pathogen

Many pathogens have global distributions, yet must also adapt to infect their hosts in specific local environments. Given that allele exchange between populations from different environments can delay or prevent adaptation through maladapted gene flow, how are local adaptations ever able to emerge and persist? Using a >1000 genome dataset of the global plant pathogen *Zymoseptoria tritici*, I am working to develop pangenome graph-based methods for genotyping structural variants to test the hypothesis that genome content and organization is under selection to preserve locally adapted genotypes (in prep). 

### Constraints and drivers of host-microbe interactions

![]({{ site.url }}{{ site.baseurl }}/images/research/lifestyle.png){: style="width: 350px; float: right; border: 25px; padding:25px" figcaption}*Many fungi shift between host-associated and free-living stages*

Microbial symbionts like bacteria and fungi are typically studied during interactions with their plant and animal hosts. Yet many microbes spend their lives in both host and non-host environments. Given that adaptations to one environment may come at the expense of adaptations to another, this raises an evolutionary conundrum: how do microbes adapt to survive in non-host environments while also adapting to interact with their hosts? I have developed an independent system to investigate the constraints and drivers of host-associated microbial adaptation using root endophytesâ€”fungi that colonize living plant roots. Endophytes are ideal for addressing these questions because they dominate root microbiomes and often shift between free-living stages and interacting with their hosts as emerging mutualists and pathogens. Yet because many endophytes have cryptic life histories, we know little about their functional impact on plant individuals, populations and communities. One common assumption based on the observation of macro-organisms is that closely related fungi are functionally redundant. I tested this assumption by sampling populations of two closely related lineages of root endophytes from natural forests and measuring differences in their multivariate response to host chemistry. I found that lineages differed in response to certain host metabolites, but that the evolution of specific responses is not constrained by overall metabolic trait architecture, underscoring the modular evolution of plant-fungal interactions (in prep). I am now investigating the genomic bases of these differences in chemical phenotypes, focusing on the role of genomic architecture in mediating metabolic trait diversification. 

### Supergenes and the evolution of chemical ecology

![]({{ site.url }}{{ site.baseurl }}/images/research/scoPathways.png){: style="width: 350px; float: right; border: 25px; padding:25px"}

Specialized metabolic pathways are critical mediators of species interactions. For example, bacteria and fungi produce and degrade myriad chemicals that act as signals, deterrents and toxins promoting and inhibiting symbiosis. Characterizing metabolic traits and the means through which they evolve therefore provides novel insight into ecological adaptation. However, connecting genotypes to chemical phenotypes is notoriously difficult. A curious feature of many specialized pathways is that the genes encoding these traits are often co-localized in genomes as ['supergenes' or metabolic gene clusters](https://www.sciencedirect.com/science/article/pii/S0959437X19300450). For my PhD, I leveraged this insight to develop novel cluster detection algorithms for investigating the genetic bases of specialized pathways, like [the degradation of plant defense compounds](https://www.nature.com/articles/s41396-018-0075-3), [the synthesis of secondary metabolites](https://academic.oup.com/mbe/article/37/10/2838/5839751), and [the production of fungal psychoactive compounds](https://www.sciencedirect.com/science/article/pii/S1754504819300352#!). Through additional molecular work, I found evidence that [fungal adaptation to plant defense compounds occurs primarily through convergent cluster assembly](https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.14943), as opposed to novel enzymatic functions, highlighting the importance of combinatorial evolution in shaping host-microbe interactions.

